routing/transit_graph_loader.cpp

```diff
@@ -41,16 +61,16 @@ TransitGraph & TransitGraphLoader::GetTransitGraph(NumMwmId numMwmId, IndexGraph
   return *(emplaceRes.first)->second;
 }
 
-unique_ptr<TransitGraph> TransitGraphLoader::CreateTransitGraph(NumMwmId numMwmId,
-                                                                IndexGraph & indexGraph) const
+unique_ptr<TransitGraph> TransitGraphLoaderImpl::CreateTransitGraph(NumMwmId numMwmId,
+                                                                    IndexGraph & indexGraph) const
 {
   platform::CountryFile const & file = m_numMwmIds->GetFile(numMwmId);
   MwmSet::MwmHandle handle = m_index.GetMwmHandleByCountryFile(file);
   if (!handle.IsAlive())
     MYTHROW(RoutingException, ("Can't get mwm handle for", file));
 
   my::Timer timer;
-  auto graphPtr = make_unique<TransitGraph>(numMwmId, m_estimator);
+  auto graphPtr = my::make_unique<TransitGraph>(numMwmId, m_estimator);
```

Comment:
Is it possible to rename graphPtr to graph?

Reply:
done